packages/testsweets/lib/src/extensions/widget_description_extension.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:testsweets/src/constants/app_constants.dart';
+import 'package:testsweets/testsweets.dart';
+
+extension WidgetDescriptionUtils on WidgetDescription {
+  double responsiveWidth(double currentScreenWidth) =>
```

Comment:
Could we update the names please. Since in the future we will actually also store the widgets bounding box with width and height we need to make it clear that this is not for the width. This is for the X position. 

Please update to `responsiveXPosition` and `responsiveYPosition` to it's clearer.

Reply:
Done